app/use_cases/accounts/apps/evolution_apis/instance/sessions_refresh_status_job.rb

```diff
@@ -5,7 +5,7 @@ class Accounts::Apps::EvolutionApis::Instance::SessionsRefreshStatusJob < Applic
 
   def perform
     Apps::EvolutionApi.connected.find_each do |evolution_api|
-      Accounts::Apps::EvolutionApis::Instance::RefreshStatus.new(evolution_api).call
+      Accounts::Apps::EvolutionApis::Instance::Delete.call(evolution_api)
```

Comment:
Talvez esse nome fica mais explicito 
```suggestion
      Accounts::Apps::EvolutionApis::Instance::DeleteInactive.call(evolution_api)
```

Reply:
Boa, mudei para `DeleteDisconnected`.